src/streamlink_cli/argparser.py

```diff
@@ -275,12 +276,14 @@ def build_parser():
     general.add_argument(
         "-l", "--loglevel",
         metavar="LEVEL",
+        choices=logger.levels,
         default="info",
         help="""
         Set the log message threshold.
-
-        Valid levels are: none, error, warning, info, debug
-        """
+        Can be repeated to load multiple files, in which case
+        the options are merged on top of each other where the
+        last config has highest priority.
+    """
```

Comment:
This needs to be indented 4 spaces further.

Reply:
I also wonder if we could potentially make what exactly this does a little more clear in terms of the documentation? I don't really see in what case you would run multiple log levels so the modified description makes it a bit more confusing, and also removes information regarding what the valid levels are.

Reply:
Seems to have got messed up when I rebased before.